server/verify.go

```diff
@@ -49,8 +49,8 @@ type VerifyOpts struct {
 
 // TODO: Change int64 fields to uint64 when compatible with ASN1 parsing.
 type gceSecurityProperties struct {
-	SecurityVersion int64 `asn1:"optional"`
-	IsProduction    bool  `asn1:"optional"`
+	SecurityVersion int64 `asn1:"explicit"`
+	IsProduction    bool  `asn1:"explicit,tag:1"`
```

Comment:
Should these also be explicit and optional?

Reply:
added